src/native/libs/System.Security.Cryptography.Native/entrypoints.c

```diff
@@ -246,6 +247,13 @@ static const Entry s_cryptoNative[] =
     DllImportEntry(CryptoNative_LoadPrivateKeyFromEngine)
     DllImportEntry(CryptoNative_LoadPublicKeyFromEngine)
     DllImportEntry(CryptoNative_LookupFriendlyNameByOid)
+    DllImportEntry(CryptoNative_IsSignatureAlgorithmAvailable)
```

Comment:
We try to keep these alpha sorted. `IsSignatureAlgorithmAvailable` should come after `CryptoNative_HmacUpdate`